test/binaryen.js/simd.js

```diff
@@ -1,5 +1,11 @@
 var module = new Binaryen.Module();
 
-var expr = module.v128.const([1, 0, 0, 0, 2, 0, 0, 0, 3, 0, 0, 0, 4, 0, 0, 0]);
-var info = Binaryen.getExpressionInfo(expr);
-console.log("v128.const i8x16 0x" + info.value.map(b => b.toString(16)).join(" 0x"));
+module.enableFeature(Binaryen.FeatureSIMD);
+
+var signature = module.addFunctionType("V", Binaryen.v128, []);
+module.addFunction("main", signature, [], module.block("", [
+  module.v128.const([1, 0, 0, 0, 2, 0, 0, 0, 3, 0, 0, 0, 4, 0, 0, 0])
+], Binaryen.v128));
+
+if (!module.validate()) throw 'did not validate :(';
```

Comment:
It would be better to print the validation results rather than throwing so the test can keep running. You could also validate before and after setting the proper features.